pyproject.toml

```diff
@@ -1,15 +1,58 @@
-[build-system]
-requires = ["setuptools<72.2", "wheel", "numpy>=1.15", "Cython>=0.29"]
+[project]
+name = "pymoo"
+description = "Multi-Objective Optimization in Python"
+authors = [{ name = "Julian Blank", email = "blankjul@outlook.com" }]
+readme = "README.rst"
+license = "Apache-2.0"
+classifiers = [
+    'Intended Audience :: Developers',
+    'Intended Audience :: Science/Research',
+    'Operating System :: OS Independent',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
+    'Programming Language :: Python :: 3.12',
+    'Programming Language :: Python :: 3.13',
+    'Topic :: Scientific/Engineering',
+    'Topic :: Scientific/Engineering :: Artificial Intelligence',
+    'Topic :: Scientific/Engineering :: Mathematics',
+]
+keywords = ["optimization"]
+dynamic = ["version"]
+urls = { homepage = "https://pymoo.org" }
+requires-python = ">= 3.9"
+dependencies = [
+    "numpy>=1.19.3",
+    "scipy>=1.1",
+    "matplotlib>=3",
+    "autograd>=1.4",
+    "cma>=3.2.2",
+    "alive-progress",
+    "dill",
+    "Deprecated",
+]
 
-[tool.cibuildwheel]
-build-verbosity = 1
-before-all = "uname -a"
```

Comment:
Since in the new `build.yml` we're running cibuildwheel as an action, it is now fully configured in the github workflow.